modules/openCypher-in-gsql/pages/openCypher-in-gsql.adoc

```diff
@@ -129,67 +117,93 @@ See below for restrictions.
 
 |atan2() |Trigonometric Returns the arctangent2 of a set of coordinates in radians.
 
+|avg() |Aggregating Returns the average of a set of values.
```

Comment:
We should unify the description of these functions with latest `openCypher` language reference's language. The latest I can find is this [ref](https://s3.amazonaws.com/artifacts.opencypher.org/openCypher9.pdf) 
For example: `avg()`'s description is `returns the average of a set of numeric values` in above reference.

If you have newer version of openCypher language reference, then use that newest.

Reply:
Great catch! 

I moved `avg` from the section of unsupported functions to section of supported functions, but didn't cross check it with OpenCypher document. Just updated the description to include the updated description.

Reply:
I think in the future we may also want to group the OpenCypher functions (for example, group aggregating functions, scalar functions, and others into different sections). Though I would do that later because I think other people are also working on improving the overall formatting.